src/pages/journal/journal.ts

```diff
@@ -171,6 +207,70 @@ export class JournalPage extends BasePageComponent implements OnInit {
     }
   }
 
+  /**selectedDate
+   * If state contains no cached data and search hasn't
+   * previously returned results then get users search results
+   * otherwise get create slot data directly
+   * @param emission
+   */
+  generateSlotAndSearchResults = (emission: SlotItem[]) => {
+    if (!this.isCachedTests && this.searchResultsAppRefs.length === 0) {
+      this.getSearchResults(emission);
+    }
+    this.createSlots(emission, this.searchResultsAppRefs);
+  }
+
+  /**
+   * Call endpoint to obtain users previously completed tests and
+   * pass to createSlots function for evaluation
+   * @param emission
+   */
+  getSearchResults = (emission: SlotItem[]) => {
+    const advancedSearchParams: AdvancedSearchParams = {
+      startDate: moment().subtract(14, 'days').format('YYYY-MM-DD'),
+      endDate: moment().format('YYYY-MM-DD'),
```

Comment:
We should read the values from config to create this time window (`journal.numberOfDaysToView`) as they can feasibly change in the future.

Reply:
Is `journal.numberOfDaysToView` going to return the value representing navigating forwards and backwards combined? Or just one way?

Reply:
Getting the number of days from config